task/src/main/java/org/exoplatform/platform/qa/ui/task/pageobject/TasksManagement.java

```diff
@@ -98,6 +104,65 @@ public void deleteTask(String task) {
     ELEMENT_TASK_BUTTON_DELETE_OK.click();
   }
 
+  public void addCowroker(String task) {
```

Comment:
addCowroker -> addCoworker

Reply:
Parameter names should be more meaningful. In this case, I think it is the text of the task, so it should be taskText or taskContent.
This remarks valid for all the others methods.

Reply:
Done